R/data.R

```diff
@@ -0,0 +1,11 @@
+#' IS Dataset
```

Comment:
@bms63 - in admiral core we include the below in the data.R file, is it necessary to request the same for admiralvaccine? 
#' @keywords datasets
#' @family datasets
#' @source

Reply:
I will add in the keywords and datasets.  They are just creating the data themselves.  Not sure how to document.  But I will make an issue to get the code that made it.